cmd/libs/go2idl/.import-restrictions

```diff
@@ -4,7 +4,8 @@
       "SelectorRegexp": "k8s[.]io",
       "AllowedPrefixes": [
         "k8s.io/kubernetes/cmd/libs/go2idl",
-        "k8s.io/kubernetes/third_party"
+        "k8s.io/kubernetes/third_party",
+        "k8s.io/kubernetes/pkg/util/sets"
```

Comment:
ah, and it wasn't working as you discovered. I see, thanks. :)

On Wed, Mar 23, 2016 at 11:02 AM, Chao Xu <notifications@github.com> wrote:

> In cmd/libs/go2idl/.import-restrictions
> <https://github.com/kubernetes/kubernetes/pull/22965#discussion_r57207707>
> :
>
> > @@ -4,7 +4,8 @@
> >        "SelectorRegexp": "k8s[.]io",
> >        "AllowedPrefixes": [
> >          "k8s.io/kubernetes/cmd/libs/go2idl",
> > -        "k8s.io/kubernetes/third_party"
> > +        "k8s.io/kubernetes/third_party",
> > +        "k8s.io/kubernetes/pkg/util/sets"
>
> deepcopy-gen and set-gen both import it. Need to add this line otherwise
> verify-import-boss fails.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22965/files/224df476e0be5ddbe0b285fd2ecce7a19e6a986e#r57207707>
>